indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -284,6 +284,24 @@ public String getBaseSequenceName()
     {
       return baseSequenceName;
     }
+
+    @Override
+    public String toString()
+    {
+      return "TaskGroup{" +
+             "groupId=" + groupId +
+             ", startingSequences=" + startingSequences +
+             ", unfilteredStartingSequencesForSequenceName=" + unfilteredStartingSequencesForSequenceName +
+             ", tasks=" + tasks +
+             ", minimumMessageTime=" + minimumMessageTime +
+             ", maximumMessageTime=" + maximumMessageTime +
+             ", exclusiveStartSequenceNumberPartitions=" + exclusiveStartSequenceNumberPartitions +
+             ", checkpointSequences=" + checkpointSequences +
+             ", baseSequenceName='" + baseSequenceName + '\'' +
+             ", completionTimeout=" + completionTimeout +
+             ", handoffEarly=" + handoffEarly +
+             '}';
```

Comment:
I think just including the fields `groupId` and `tasks` might be more useful for logging. The other information would make the messages too verbose.

Reply:
Similarly, the `TaskData.toString()` should stop including the `checkpointSequences` field to make the log messages cleaner.